verifyApiKey/data_test.go

```diff
@@ -183,18 +184,5 @@ var _ = Describe("DataTest", func() {
 			Expect(len(apiProducts)).Should(BeEquivalentTo(0))
 
 		})
-
-		It("should get kms attributes", func() {
```

Comment:
Is this covered some where?
With this PR 0.7% down on `github.com/apid/apidApiMetadata/verifyApiKey` code coverage

Reply:
package `common`, because that's where the `GetKmsAttributes` code is.